include/fox/inplace_free_list.hpp

```diff
@@ -108,6 +113,19 @@ namespace fox
 			return first_free_ == offset_type_npos;
 		}
 
+		[[nodiscard]] bool is_sorted() const noexcept
+		{
+			offset_type current = first_free_;
+			bool is_sorted_ = true;
+			while (current != offset_type_npos && is_sorted_) 
+			{
+				offset_type next = reinterpret_cast<const offset_accessor*>(std::data(storage_))[current].offset;
+				is_sorted_ = is_sorted_ && (next > current);
```

Comment:
Get rid of `is_sorted_` and short-circuit this. Also naming convention, variables with trailing underscore are member variables.
```cpp
if(next < current)
    return false;
```